src/request/create/CreateLow.js

```diff
@@ -46,17 +46,17 @@ class CreateLow {
         this._pinInput.on(
             PinInput.Events.PIN_ENTERED,
             /** @param {number} pin */ async pin => {
-                $confirmMessage.classList.add('hidden');
-                $notMatchingMessage.classList.add('hidden');
+                $confirmMessage.classList.add('display-none');
```

Comment:
I believe the `hidden` class is used here on purpose to not make the content jump when unhiding it.

Reply:
To keep the no-jumping behaviour the message is absolutely positioned in an always visible container.

I changed the visibility:hidden to display:none two avoid that:
- both messages take space in the layout while at most one is visible at the same time
- the messages are not rendered at the same position
- the bottom message has no margin to the pinpad dots